### ISSUE TITLE
Request for a simple user guide for a novice user

### ISSUE THREAD

**Issue by prabhuMRaj** [forsti0506/a11y-sitechecker#271]
Hi,

Is there any simple user guide for a blackbpox tester (who has limited to no coding knowledge but can follow the user guide to make the full use of the application) to use this to generate an accessibility report, please?

Could you help with this, please?

**Comment by forsti0506** [comment]
I will think about it, because I'm not sure if it is the target group for me, to use it with no coding skills! Please give me a few days!

**Comment by prabhuMRaj** [comment]
> I will think about it, because I'm not sure if it is the target group for me, to use it with no coding skills! Please give me a few days!

Thank you! its much appreciated. Looking forward to hearing from you.

**Comment by prabhuMRaj** [comment]
> I will think about it, because I'm not sure if it is the target group for me, to use it with no coding skills! Please give me a few days!

HI, have you thought about this, please? would appreciate if you can help in this regard.

**Comment by deligant** [comment]
Hello Martin: I think the readme.md is missing some important part between "install" and "Usage": after installing there is just a node_modules directory and launching the command on the CLI givrs a "command not found" answer...
I even cloned the repo, run a npm i --legacy-peer-deps as the regular npm i isn't working...
Thank you in advance for some hint on how to use it from the CLI
